CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+## Publishing
+
+Make sure your repo is up to date with the latest master from upstream.
+To publish a new minor release to NPM:
+
+```sh
+npm version minor
```

Comment:
I think this is pretty clear from the npm man page. No need to reference some random article, it might become outdated (the one you mentioned is over a year old already).